assets/eip-5606/ERC721Full.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
[Editorial] If you are author or copyright owner of this piece of code, please grant it in CC0 license. Otherwise, please remove it from this repo per https://github.com/ethereum/EIPs/blob/master/LICENSE.md.
here and everywhere else.